skbeam/core/correlation.py

```diff
@@ -190,18 +191,22 @@ def _init_state(num_levels, num_bufs, labels):
         past_intensity,
         future_intensity,
         img_per_level,
-        label_mask,
+        label_array,
         track_level,
         cur,
         pixel_list,
-        label_mapping,
+        num_pixels,
+        lag_steps,
```

Comment:
I am trying to stop repeating things use for both 1 and 2 correlation. therefore, I put num_pixels and lag_steps inside _input function and added it here.